scio-test/src/test/scala/com/spotify/scio/sql/BeamSQLTest.scala

```diff
@@ -34,6 +34,14 @@ import org.scalatest.Assertion
 import scala.collection.JavaConverters._
 import com.spotify.scio.avro
 
+class IsOver18UdfFn extends SerializableFunction[Integer, Boolean] {
```

Comment:
wondering why move outside of `TestData`? Could we move the hole `TestData` outside of `BeamSQLTest`?

since we have more UDF/s inside TestData ... this seems a bit weird?

Reply:
I was just testing if this it what makes the compiler crash. Apparently not...

Reply:
@jto yeah I realized this PR as a bunch of test stuff after reading your comments on slack ... all makes sense now 😄

Reply:
@jto can we move back these classes?